sympy/core/power.py

```diff
@@ -440,6 +440,9 @@ def _eval_is_positive(self):
         elif self.base.is_nonpositive:
             if self.exp.is_odd:
                 return False
+            elif self.base.is_zero:
+                if self.exp.is_real and (self.exp.is_zero is False):
```

Comment:
can this be
```
if self.exp.is_real:
    return self.exp.is_zero  # 0**0->True, 0**(pos or neg) -> (False), 0**unknown -> unknown
```

Reply:
Yeah, this can be done. Just the `0**0` case is already handled in the first if-clause and both structures have no significant difference.

Reply:
Also I can't replicate this with `_eval_is_negative`'s code. I am still trying to understand the code in core modules and I don't want to break anything until I understand it.